downloads/swan-lake-release-notes/swan-lake-beta2/RELEASE_NOTE.md

```diff
@@ -1,42 +1,217 @@
+    RecursiveType b = [1, []];
+    RecursiveType c = [1, [a]];
+}
+```
+
+- Changed the static type of the string iteration from `string` to `string:Char`.
+
+```ballerina
+public function main() {
+    string str = "foo";
+
+    foreach string:Char s in str {
+        io:println(s);        
+    }
+
+    record {| string:Char value; |}? next = str.iterator().next();
+
+    if !(next is ()) {
+        io:println(next.value);
+    }
+}
+```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+## Runtime Updates
+
+### Improvements
+
+- Introduced the support for `configurable` variables of union types via the TOML syntax. 
+
+```ballerina
+configurable map<anydata> myMap = ?;
+configurable int|string id = ?;
+```
+
+**TOML:**
+
+```toml
+id = "12345"
+
+[myMap]
+name = "John"
+age = 10
+```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `postgresql` Package
+
+Introduced the new `postgresql` database package with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.
+
+#### `log` Package
+
+Added observability span context values to log messages when observability is enabled.
+
+#### `io` Package
+
+Introduced the `io:fprint` and `io:fprintln` APIs which allows the users to print content to a given stream (standard error or standard out).
+
+```ballerina
+io:fprint(io:stderr, "Unexpected error occurred");
+io:fprintln(io:stderr, "Unexpected error occurred");
+io:fprint(io:stdout, "Passed without an error");
+io:fprintln(io:stdout, "Passed without an error");
+```
+
+#### `websocket` Package
+
+Introduced the declarative auth support for the server-side. 
+
+#### `websub` Package
+
+Added a utility method to retrieve HTTP headers from the `websub:ContentDistributionMessage`.
+
+### Improvements
+
+#### Database Packages
+
+Made the return type of the stream to be inferred now as a second parameter to the query remote method.
+
+**Previous Syntax**
+
+```ballerina
+stream<record{}, error> resultStream =
+        jdbcClient->query(`SELECT * FROM Customers`, Customer);
+
+stream<Customer, sql:Error> customerStream =
+        <stream<Customer, sql:Error>>resultStream;
+```
+
+According to the previous syntax, you can give the `Customer` record as the expected record type of the stream but you can not get that stream directly. For that, you have to cast the returned stream to the expected stream. Therefore, this has been improved to directly get data as an expected record type.
```

Comment:
Have we used second person/ 'you' in release notes before?

Reply:
Yes. That is the practice followed throughout the docs.